content/documentation/pages/5-feature-guides/1-streams/11-monitoring.md

```diff
@@ -6,39 +6,39 @@ description: 'Monitoring streaming data pipelines with Prometheus and InfluxDB'
+- `stream.name`: The name of the Stream that contains the applications that send the metrics
+- `application.name`: The name or label of the application that reports the metrics
+- `application.type`: The type (Source, Processor, or Sink) of the application that reports the metrics
+- `application.guid`: Unique instance identifier of the application instance that reports the metrics
+- `application.index`: Application instance ID (when available)
 
-If the Data Flow server is started with the spring.cloud.dataflow.grafana-info.url property pointing to your Grafana URL, then the Grafana feature is enabled and Data Flow UI will provide you with Grafana-buttons that can open particular dashboard for given stream, application or application instance. Following screenshot illustrates these buttons:
+If the Data Flow server is started with the `spring.cloud.dataflow.grafana-info.url` property pointing to your Grafana URL, the Grafana feature is enabled and Data Flow UI provides you with Grafana-buttons that can open a particular dashboard for a given stream, application, or application instance. The following screenshots illustrate these buttons:
 
 ![Stream List Monitoring](images/grafana-scdf-ui-buttons-apps.png)
 
 ![Runtime Applications Monitoring](images/grafana-scdf-ui-buttons-streams.png)
 
-As setting up Prometheus and InfluxDB is different depending on the platform you are running on, we provide instructions for each platform. In Spring Cloud Data Flow 2.x, local server and Kubernetes instructions have been provided.
+As setting up Prometheus and InfluxDB is different depending on the platform on which you run, we provide instructions for each platform. In Spring Cloud Data Flow 2.x, local server and Kubernetes instructions have been provided.
 
 ## Local
 
+This section describes how to set up Prometheus and InfluxDB for a local machine.
+
 ### Prometheus
 
-Prometheus is a popular pull based time series database that pulls the metrics from the target applications from preconfigured endpoints. Prometheus needs to be regularly provided with the URLs of the target applications to monitor. This task is delegated to a component called Service Discovery, which usually a platform specific implementation. The Data Flow server provides a standalone Service Discovery service that uses the Data Flow /runtime/apps REST endpoint to compute the URL locations of the deployed application’s Prometheus endpoints.
+Prometheus is a popular pull-based time series database that pulls the metrics from the target applications from preconfigured endpoints. Prometheus needs to be regularly provided with the URLs of the target applications to monitor. This task is delegated to a component called Service Discovery, which is usually a platform-specific implementation. The Data Flow server provides a standalone Service Discovery service that uses the Data Flow `/runtime/apps` REST endpoint to compute the URL locations of the deployed application’s Prometheus endpoints.
```

Comment:
Spring Cloud Data Flow instead of Data Flow

Reply:
Yup.